Library/ViewModels/CommentsViewModelTests.swift

```diff
@@ -314,27 +312,27 @@ internal final class CommentsViewModelTests: TestCase {
         self.loadCommentsAndProjectIntoDataSourceComments
           .assertValues([envelope.comments], "No new comments are emitted")
         self.loadCommentsAndProjectIntoDataSourceProject
-          .assertValues([project], "No new projects are emitted")
+          .assertValues([.template], "No new projects are emitted")
 
         self.scheduler.advance()
 
         self.loadCommentsAndProjectIntoDataSourceComments
           .assertValues([envelope.comments, updatedEnvelope.comments], "New comments are emitted")
         self.loadCommentsAndProjectIntoDataSourceProject
-          .assertValues([project, project], "Same project is emitted again")
+          .assertValues([.template, .template], "Same project is emitted again")
       }
     }
   }
 
-  func testProjectPagination_WhenLimitReached_CommentsAreUpdatedInDataSource() {
+  func testProjectPagination_WhenLimitReached_CommentsAreUpdatedInDataSource_AndEmitsLoadMoreIndicatorValue() {
```

Comment:
the names of these functions are a bit much imo. We should simplify it something like:

`testLoadCommentsAndLoadingIndicator_WillDisplayRowAtEnd`

The exact name is open for discussion but we should either simplify the name or move some of the tests out into a separate method.